python_modules/dagster/dagster_tests/asset_defs_tests/partition_mapping_tests/test_time_window_partition_mapping.py

```diff
@@ -638,3 +638,60 @@ def test_downstream_partition_has_valid_upstream_partitions(
     )
     assert result.partitions_subset.get_partition_keys() == valid_partitions_mapped_to
     assert result.required_but_nonexistent_partition_keys == required_but_nonexistent_partition_keys
+
+
+@pytest.mark.parametrize(
+    "partition_key,expected_upstream_partition_key,expected_downstream_partition_key",
+    [
+        (
+            "2023-11-04",
+            "2023-11-03",
+            "2023-11-05",
```

Comment:
could also include a fall/spring test case here that doesn't navigate a transition just to make sure it works correctly in the expected case